spec/models/comment_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Comment do
+  context "Validate" do
+    it 'should have a valid factory' do
+      FactoryGirl.create(:comment)
```

Comment:
@phendych contohnya salah berarti, contoh ini emang lumayan banyak salah soalnya bikinnya lumayan buru2 dulu, so don't trust the examples :)